nestjs/src/certificates/certificates.service.ts

```diff
@@ -82,4 +82,16 @@ export class CertificationsService {
       },
     };
   }
+
+  public async removeCertificate(studentId: number) {
+    const certificate = await this.certificateRepository.findOne({
+      where: { studentId },
+    });
+
+    if (!certificate) {
+      throw new NotFoundException('Certificate not found');
+    }
+
+    await this.certificateRepository.remove(certificate);
+  }
```

Comment:
@aliakseitokarev API that I mentioned is ready, please add `DELETE` request to `{configService.awsServices.restApiUrl}/certificate/{certificate.s3Key}` to your solution